bin/Decompile/Main.cpp

```diff
@@ -229,9 +229,13 @@ int main(int argc, char *argv[]) {
 
   spec.ForEachFunction([&lifter, &names, &target_funcs](auto decl) {
     llvm::Function *func;
-    if (target_funcs.empty() ||
+    if (FLAGS_lift_list.empty() ||
         target_funcs.find(decl->address) != target_funcs.end()) {
+      LOG(INFO) << "attempting to lift: " << std::hex << decl->address;
       func = lifter.LiftEntity(*decl);
+      if (!func) {
+        LOG(ERROR) << "Failed to lift: " << std::hex << decl->address;
```

Comment:
`DLOG`

Reply:
Since this is a binary not a lib, logging the error is probably ideal